tests/test_connectives.py

```diff
@@ -0,0 +1,94 @@
+from pyscipopt import Model
+
+################################################################################
+#
+# Testing AND/OR constraints
+#   check whether any error is raised
+# see http://scip.zib.de/doc-5.0.1/html/cons__and_8c.php
+#   for resultant and operators definition
+# CAVEAT: ONLY binary variables are allowed
+#   Integer and continous variables behave unexpectedly (due to SCIP?)
+# TBI: automatic assertion of expected resultant VS optimal resultant
+#   (visual inspection at the moment)
+# TBI: implement and test XOR constraint
+#
+################################################################################
+
+### AUXILIARY ###
+def setModel(vtype="B", name=None, imax=2):
+    if name is None: name = "model"
+    m = Model(name)
+    m.hideOutput()
+    i = 0
+    m.addVar("r", vtype)
+    while i < imax:
+        m.addVar("v%s" % i, vtype)
+        i+=1
+    return m
+
+def getVarByName(m, name):
+    try:
+        return [v for v in m.getVars() if name == v.name][0]
+    except IndexError:
+        return None
+
+def getAllVarsByName(m, name):
+    try:
+        return [v for v in m.getVars() if name in v.name]
+    except IndexError:
+        return []
+
+
+def setConss(m, vtype="B", val=0, imax=1):
+    i = 0
+    while i < imax:
+        vi = getVarByName(m,"v%s" % i)
+        m.addCons(vi == val, vtype)
+        i+=1
+    return
+
+def printOutput(m):
+    status = m.getStatus()
+    r = getVarByName(m,"r")
+    rstr = "%d" % round(m.getVal(r))
+    vs = getAllVarsByName(m, "v")
+    vsstr = "".join(["%d" % round(m.getVal(v)) for v in vs])
+    print("Status: %s, resultant: %s, operators: %s" % (status, rstr, vsstr))
+
+### TEST ###
+def test_connective(m, connective, sense="min"):
```

Comment:
You cannot define a function that's called `test_...()` and pass some parameters, as these functions will be called by py.test directly, so the parameters are undefined.